backend/database/database.go

```diff
@@ -322,13 +322,14 @@ func (d *Database) UserIDByGoogleID(googleID string) (uuid.UUID, error) {
 }
 
 type Review struct {
-	ID         uuid.UUID
-	Text       string
-	VendorID   uuid.UUID
-	UserID     uuid.UUID
-	DatePosted time.Time
-	StarRating *int
-	ReplyTo    *uuid.UUID
+	ID             uuid.UUID
+	Text           string
+	VendorID       uuid.UUID
+	UserID         uuid.UUID
+	DatePosted     time.Time
+	StarRating     *int
+	ReplyTo        *uuid.UUID
+	VendorFavorite *int
```

Comment:
Could you change the type of `VendorFavorite` to `bool`? It will make it a lot easier to work with this variable if it has the correct type.